tests/cli/commands/test_variable_command.py

```diff
@@ -31,58 +32,15 @@ def setUpClass(cls):
         cls.dagbag = models.DagBag(include_examples=True)
         cls.parser = cli_parser.get_parser()
 
```

Comment:
```suggestion

    def setUp(self):
        clear_db_variables()

    def tearDown(self):
        clear_db_variables()

```
This will allow us to maintain isolation between tests and reduce side effects.